dockerfiles/docker-entrypoint.sh

```diff
@@ -1,62 +1,67 @@
 #!/bin/sh
 
 help() {
-	printf "Command line:\n\n"
-	printf "  -h  Display this help\n"
-	printf "  -d  Dry run to update the configuration files.\n"
-	printf "  -f  Always update on the configuration files (existing files are renamed with the .old suffix)\n"
-	printf "      Without this option, the new configuration files are copied with the .new suffix\n"
-	printf "\nEnvironment variables:\n\n"
-	printf "  INSTANCE_NAME settings.yml : general.instance_name\n"
-	printf "  AUTOCOMPLETE  settings.yml : search.autocomplete\n"
-	printf "  BASE_URL      settings.yml : server.base_url\n"
-	printf "  MORTY_URL     settings.yml : result_proxy.url\n"
-	printf "  MORTY_KEY     settings.yml : result_proxy.key\n"
-	printf "  BIND_ADDRESS  uwsgi bind to the specified TCP socket using HTTP protocol. Default value: \"${DEFAULT_BIND_ADDRESS}\"\n"
-	printf "\nVolume:\n\n"
-	printf "  /etc/searx    the docker entry point copies settings.yml and uwsgi.ini in this directory (see the -f command line option)\n"
-	echo
-	exit 0
+    cat <<EOF
+Command line:
+  -h  Display this help
+  -d  Dry run to update the configuration files.
+  -f  Always update on the configuration files (existing files are renamed with
+      the .old suffix).  Without this option, the new configuration files are
+      copied with the .new suffix
+Environment variables:
+  INSTANCE_NAME settings.yml : general.instance_name
+  AUTOCOMPLETE  settings.yml : search.autocomplete
+  BASE_URL      settings.yml : server.base_url
+  MORTY_URL     settings.yml : result_proxy.url
+  MORTY_KEY     settings.yml : result_proxy.key
+  BIND_ADDRESS  uwsgi bind to the specified TCP socket using HTTP protocol.
+                Default value: ${DEFAULT_BIND_ADDRESS}
+Volume:
+  /etc/searx    the docker entry point copies settings.yml and uwsgi.ini in
+                this directory (see the -f command line option)"
+
+EOF
 }
 
 export DEFAULT_BIND_ADDRESS="0.0.0.0:8080"
-if [ -z "${BIND_ADDRESS}" ]; then
-    export BIND_ADDRESS="${DEFAULT_BIND_ADDRESS}"
-fi
-
-# Parse special command line
-# see docs/admin/installation-docker.rst
-# display the help message without the version
-if [ "$1" = "help" ]; then
-	help
-fi
-
-# Version
-export SEARX_VERSION=$(su searx -c 'python3 -c "import six; import searx.version; six.print_(searx.version.VERSION_STRING)"' 2>/dev/null)
-printf 'searx version %s\n\n' "${SEARX_VERSION}"
+BIND_ADDRESS="${BIND_ADDRESS:-${DEFAULT_BIND_ADDRESS}}"
+export BIND_ADDRESS
```

Comment:
For my own knowledge why it can't be `export BIND_ADDRESS="${BIND_ADDRESS:-${DEFAULT_BIND_ADDRESS}}"` in one line ?

Reply:
It can, my fail.